src/main/java/com/jfb/tickets/controller/UserController.java

```diff
@@ -0,0 +1,52 @@
+package com.jfb.tickets.controller;
+
+import com.jfb.tickets.model.Ticket;
+import com.jfb.tickets.model.User;
+import com.jfb.tickets.service.UserService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.*;
+
+@RestController
+@RequestMapping("/api/users")
+public class UserController {
+    private final UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PostMapping("")
```

Comment:
just ` @PostMapping`, without arguments id the url is equal to url in `@RequestMapping`